src/app/feed/shared/connection-config.model.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2015 SUSE Linux
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE.txt file for details.
+ */
+
+import * as _ from "lodash";
```

Comment:
For some reason `import _ from "lodash";` not works correctly, but using `import * as _ from "lodash";` solves the problem.